api/app/Http/Middleware/AuthenticateWithJwtOrSanctum.php

```diff
@@ -30,6 +30,10 @@ public function handle(Request $request, Closure $next)
                 throw new AuthenticationException('Pro subscription required for API access.');
             }
 
```

Comment:
let's not mix everything, we shouldn't check this here and in the dedicated middleware